.eslintrc.js

```diff
@@ -0,0 +1,94 @@
+module.exports = {
+    root: true,
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        project: './tsconfig.json',
+        // sourceType: "module"
+    },
+    plugins: ['@typescript-eslint', 'react', 'import', 'functional', 'sonarjs'],
+    extends: [
+        'eslint:recommended',
```

Comment:
shouldn't we extend from the digitalcitizenship rules?

Reply:
I don't know.. io-app settings aren't based on digital citizenship rules, I simply cut and pasted that section

Reply:
ok I've asked in Slack if it's still relevant or not